parser/src/main/java/com/linkedin/avroutil1/util/ConfigurableAvroSchemaComparator.java

```diff
@@ -39,13 +42,27 @@ public class ConfigurableAvroSchemaComparator {
   public static boolean equals(AvroSchema a, AvroSchema b, SchemaComparisonConfiguration config) {
     validateConfig(config);
     Set<SeenPair> seen = new HashSet<>(3);
+    List<AvroSchemaDifference> differences = new ArrayList<>();
     try {
-      return equals(a, b, config, seen);
+      equals(a, b, config, seen, differences);
+      return differences.isEmpty();
     } catch (IOException e) {
       return false;
     }
   }
 
+  public static List<AvroSchemaDifference> findDifference(AvroSchema a, AvroSchema b, SchemaComparisonConfiguration config) {
+    validateConfig(config);
+    Set<SeenPair> seen = new HashSet<>(3);
+    List<AvroSchemaDifference> differences = new ArrayList<>();
+    try {
+      equals(a, b, config, seen, differences);
+    } catch (IOException ignored) {
+      // ignored
+    }
+    return differences;
+  }
```

Comment:
Also there are code duplicate between method **equals** and **findDifference**.

You can have method **equals** call method **findDifference**:

Reply:
Added a helper method which will be called by both `equals` and `findDifference` methods.